File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        res={}
        for i,num in enumerate(nums):
            compliment=target-num
            if compliment in res:
                return [res[compliment],i]
            res[num]=i
        
        return None
       

File: Data Structures & Algorithms/majority-element/submission-1.java
class Solution {
    public int majorityElement(int[] nums) {
        int n=nums.length;
        Arrays.sort(nums);
        int c=1;
        for(int i=1;i<n;i++){
            if(nums[i]==nums[i-1]){
                c++;
                if(c>n/2){
                    return nums[i];
                }
            }
            else{
                c=1;
            }
        }
        return nums[0];
        
    }
}